portfolio/src/main/webapp/index.html

```diff
@@ -4,53 +4,40 @@
     <meta charset="UTF-8">
     <title>Ryan's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script>
-
-        function addRandomFact() {
-            const facts = 
-                ['hair', 'height', 'lakers', 'hammock', 'ucsb', 'italian'];
-
-            // Pick a random fact.
-            const fact = facts[Math.floor(Math.random() * facts.length)];
-
-            // Add it to the page.
-            const showFact = document.getElementById(fact);
-            showFact.style.display = "block";
-        }
-
-    </script>
+    <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
       <h1>Ryan's Portfolio</h1>
+      <img src="images/IMG-3136.jpg" class="mediumImg"/>
```

Comment:
I believe classes should be in the form medium-img instead of lowerCamelCase

Reply:
Good to know. Thanks